editor/blocks/quote/style.scss

```diff
@@ -0,0 +1,24 @@
+#editor blockquote {
```

Comment:
I'm not the best person for naming things hahaha :) Always had trouble with it. I'll choose `editor-blocks-quote` to match the folder structure, I guess we should define a convention somewhere